game/md2.ksy

```diff
@@ -0,0 +1,194 @@
+meta:
+  id: md2
+  title: Quake II player model (version 8)
+  application: Quake II
+  file-extension: md2
+  license: CC0-1.0
+  endian: le
+doc: |
+  The MD2 format is used for 3D animated models in id Sofware's Quake II.
+
+  A model consists of named `frames`, each with the same number of `vertices`
+  (`vertices_per_frame`). Each such vertex has a `position` in model space,
+  and a `normal_index` which you must look up to get its normal. Each vertex
+  has the same topological "meaning" across frames, in terms of triangle and
+  texture info; it just varies in position and normal for animation purposes.
+
+  How the vertices form triangles is defined via disjoint `triangles` or via
+  `gl_cmds` (which allows strip and fan topology.) Each triangle contains three
+  `vertex_indices` into frame vertices, and three `tex_point_indices` into
+  global `tex_coords`. Each texture point has pixel coords `u_px` and `v_px`
+  ranging from 0 to `skin_{width,height}_px` respectively, along with
+  `{u,v}_normalized` ranging from 0 to 1 for your convenience.
+
+  A GL command has a `primitive` type (`TRIANGLE_FAN` or `TRIANGLE_STRIP`) along
+  with some `vertices`. Each GL vertex contains `tex_coords_normalized` from 0
+  to 1, and a `vertex_index` into frame vertices.
+
+  A model may also contain `skins`, which are just file paths to PCX images.
+  However, this is empty for many models, in which case it is up to the client
+  (e.g. Q2PRO) to offer skins some other way (e.g. by similar filename in the
+  current directory.)
+doc-ref: |
+  https://icculus.org/~phaethon/q3a/formats/md2-schoenblum.html
+  http://tfc.duke.free.fr/coding/md2-specs-en.html
+seq:
+  - id: magic
+    contents: IDP2
+  - id: version
+    type: u4
+    doc: Always 8, apparently.
+  - id: skin_width_px
+    type: u4
+  - id: skin_height_px
+    type: u4
+  - id: bytes_per_frame
+    type: u4
+  - id: num_skins
+    type: u4
+  - id: vertices_per_frame
+    type: u4
+  - id: num_tex_coords
+    type: u4
+  - id: num_triangles
+    type: u4
+  - id: num_gl_cmds
+    type: u4
+  - id: num_frames
+    type: u4
+  - id: ofs_skins
+    type: u4
+  - id: ofs_tex_coords
+    type: u4
+  - id: ofs_triangles
+    type: u4
+  - id: ofs_frames
+    type: u4
+  - id: ofs_gl_cmds
+    type: u4
+  - id: ofs_eof
+    type: u4
+instances:
+  skins:
+    pos: ofs_skins
+    size: 64
+    type: str
+    terminator: 0
```

Comment:
There is a shortcut for this:

```suggestion
    type: strz
```

which would be I guess _idiomatic_ to use, but I wouldn't really mind keeping the expanded version either (note: there is another `type: str` + `terminator: 0` later in the file, so if you decide to switch to `type: strz` here, please don't forget to do it over there as well).

Reply:
Changed in both locations. As long as this still reserves 16 chars and extracts the null-terminated portion, it's ok. Seems to still work in the IDE.